pycytominer/cyto_utils/DeepProfiler_processing.py

```diff
@@ -360,7 +360,32 @@ def __init__(
 
         self.deep_data = deep_data
 
-    def get_single_cells(self, output=False):
+    def get_single_cells_column(self, output=False, column_num=0):
```

Comment:
hmm, can you update the name of this function? Based on the functionality I'm seeing, it looks like your doing more than "getting a single cells column".

Also, how is this functionality different than `get_single_cells()`?

Reply:
Maybe this function was added in error?

Reply:
Yep, the `get_single_cells_column` was added in error. Sorry about that! Deleted as of https://github.com/cytomining/pycytominer/pull/218/commits/04e518c54c1ba7aeaab34dadd519158bb3c066d1